ledger-service/http-json/src/main/scala/com/digitalasset/http/util/ExceptionOps.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.http.util
+
+object ExceptionOps {
+
+  implicit class ExceptionOps(val underlying: Throwable) extends AnyVal {
+    def description: String = getDescription(underlying)
+  }
+
+  def getDescription(e: Throwable): String = {
+    val name: String = getClass.getName
+    Option(e.getMessage).filter(_.nonEmpty) match {
```

Comment:
Is there a specific reason why you'd always prepend the class name? Why not doing that only if the message is missing? If you need to prepend the class name you can do it at call site.

Reply:
I actually want to provide the context, the exception class name. I could just use `Throwable#toString` but anyone can override id, so I don't really control it. with `getDescription` I control exactly how I want to report exceptions. If exception class name makes too much noise we could change it later, but I believe it will be useful for DABL team, they extend JSON API.